iterators.tex

```diff
@@ -7346,23 +7352,287 @@
 
 \indexlibrary{\idxcode{crbegin(const C\& c)}}%
 \begin{itemdecl}
-template <@\oldtxt{class}\newtxt{_Auto}@ C> auto crbegin(const C& c) -> decltype(@\oldoldtxt{std::}\newtxt{ranges\oldtxt{_v1}::}@rbegin(c));
+template <class C> auto crbegin(const C& c) -> decltype(std::rbegin(c));
 \end{itemdecl}
 \begin{itemdescr}
-\pnum \returns \tcode{\oldoldtxt{std::}\newtxt{ranges\oldtxt{_v1}::}rbegin(c)}.
+\pnum \returns \tcode{std::rbegin(c)}.
 \end{itemdescr}
 
 \indexlibrary{\idxcode{crend(const C\& c)}}%
 \begin{itemdecl}
-template <@\oldtxt{class}\newtxt{_Auto}@ C> auto crend(const C& c) -> decltype(@\oldoldtxt{std::}\newtxt{ranges\oldtxt{_v1}::}@rend(c));
+template <class C> auto crend(const C& c) -> decltype(std::rend(c));
 \end{itemdecl}
 \begin{itemdescr}
-\pnum \returns \tcode{\oldoldtxt{std::}\newtxt{ranges\oldtxt{_v1}::}rend(c)}.
+\pnum \returns \tcode{std::rend(c)}.
 \end{itemdescr}
+}
+
+{\color{newclr}
+\rSec2[iterator.range.begin]{\tcode{begin}}
+\pnum
+The name \tcode{begin} denotes a customization point object
+(\ref{customization.point.object}). The expression
+\tcode{ranges::begin(E)} for a subexpression \tcode{E} has the same
+effects as:
+
+\begin{itemize}
+\item
+  \tcode{(E).begin()} if \tcode{E} has type
+  \tcode{std::initializer_list<T>} for some type \tcode{T}.
```

Comment:
Why? It's equally unsafe, is it not? Backing storage for an `initializer_list` is on the stack IIUC.